src/vivarium_gates_mncnh/constants/scenarios.py

```diff
@@ -1,29 +1,36 @@
+from dataclasses import dataclass
 from typing import NamedTuple
 
 #############
 # Scenarios #
 #############
 
 
+@dataclass
 class InterventionScenario:
-    def __init__(
-        self,
-        name: str,
-        # todo add additional interventions
-        # has_treatment_one: bool = False,
-        # has_treatment_two: bool = False,
-    ):
-        self.name = name
-        # self.has_treatment_one = has_treatment_one
-        # self.has_treatment_two = has_treatment_two
+    name: str
+    bemonc_cpap_access: str = "baseline"
+    cemonc_cpap_access: str = "baseline"
 
 
 class __InterventionScenarios(NamedTuple):
     BASELINE: InterventionScenario = InterventionScenario("baseline")
     # todo add additional intervention scenarios
-
-    def __get_item__(self, item):
-        return self._asdict()[item]
+    FULL_CPAP_BEMONC: InterventionScenario = InterventionScenario(
+        "full_cpap_bemonc", "full", "baseline"
```

Comment:
Nit: this is more readable if you use keyword arguments - name doesn't need to be a kwarg imo though